test/allCommands.nut.ts

```diff
@@ -75,6 +79,7 @@ describe('verifies all commands run successfully ', () => {
       'lastname',
       'timezonesidkey'
     );
+    expect(output.result.fields.id).to.not.equal(mainUserId);
```

Comment:
should we also be verifying that it equals whatever is printed or whatever it should be?

Reply:
the NUT will generate a user with an auto-generated ID.  Are you wanting to assert something about the length and first 3 characters?

This assertion is really just that the bug is fixed.

Reply:
Looks like the unit test covers that nicely as well.  👍